breakglass.go

```diff
@@ -90,10 +119,16 @@ func main() {
 
 	signer, err := loadHostKey(*hostKeyPath)
 	if err != nil {
-		if os.IsNotExist(err) {
-			log.Printf("see https://github.com/gokrazy/breakglass#installation")
+		if !os.IsNotExist(err) {
+			log.Fatalf("see https://github.com/gokrazy/breakglass#installation")
```

Comment:
This conditional doesn’t make sense to me: the installation guide won’t help in case there are other errors than “file not found”. Why not just remove the conditional entirely?

Reply:
@stapelberg sorry for the delay. I've resolved the comment. If any error occurs it will still be logged now.